src/util/Random.h

```diff
@@ -104,6 +106,24 @@ class RandomDoubleGenerator {
   std::uniform_real_distribution<double> _distribution;
 };
 
+// GENERATE UUID
+class UuidGenerator {
+ public:
+  explicit UuidGenerator(
+      RandomSeed seed = RandomSeed::make(std::random_device{}()))
+      : randomEngine_{seed.get()}, gen_(randomEngine_) {}
+
+  std::string_view operator()() {
```

Comment:
I think this can simply directly return the `std::string` without storing it. There is no way to access it after a single retrieval anyway.